### PR TITLE
Create blog post on OpenTelemetry at Kubecon NA

### PR DESCRIPTION
**Pull request by alolita**
Blog post on OpenTelemetry Talks and Activities at Kubecon NA 2022 for publication as soon as approved.

### CODE REVIEW

**Comment by cartermp** [comment]
I say let's schedule this for early next week, assuming it's approved.

**Comment by chalin** [comment]
Make a review pass now.

File: content/en/blog/2022/otel-at-kubecon-na-2022/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: "Join us for OpenTelemetry Talks and Activities at Kubecon NA 2022"
+linkTitle: Kubecon NA 2022
+date: 2022-10-06
+author: "[Alolita Sharma](https://github.com/alolita)"
+---
+
+The OpenTelemetry project maintainers, and members of the governance committee and technical committee are excited to be at [Kubecon NA](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/) in a few weeks! Join in to meet up in person or virtually for [OpenTelemetry](https://opentelemetry.io) activities in Detroit from October 24 - 28, 2022.
```
[the remark this answers is not in the document]
Comment (chalin) [reply content/en/blog/2022/otel-at-kubecon-na-2022/index.md]: Fixed

**Comment by chalin** [comment]
FYI, in my last commit I renamed the file: we don't need a folder for the post since it doesn't have any local resources, and the name doesn't need 2022 in it since this post is under the 2022 folder.

File: content/en/blog/2022/kubecon-na.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+[Open Observability Day](https://events.linuxfoundation.org/open-observability-day-north-america/)
+has several OpenTelemetry sessions including panel discussions, workshops,
+individual talks and lightning talks. This event will be on Tuesday Oct 25, 2022
+and is co-located with Kubecon NA. The schedule of talks can be found here.
+
+[eBPF Day](https://events.linuxfoundation.org/cloud-native-ebpf-day-north-america/)
+also has some OpenTelemetry talks focused on profiling and tracing using eBPF
+events. This event will be held on Tuesday, Oct 25 2022 and is co-located with
+Kubecon NA. Check out the event schedule
+[here](https://events.linuxfoundation.org/cloud-native-ebpf-day-north-america/program/schedule/).
```
Comment (chalin) [inline content/en/blog/2022/kubecon-na.md:51]:
```suggestion
Kubecon NA. For details, check out the
[schedule](https://events.linuxfoundation.org/cloud-native-ebpf-day-north-america/program/schedule/).
```

File: content/en/blog/2022/kubecon-na.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+## Come Visit the OpenTelemetry Project Booth
+
+Drop by and say Hi! at the OpenTelemetry project booth in the Kubecon NA Project
+Pavilion. If you’re lucky, you may even pick up some OpenTelemetry swag!
+
+The pavilion hours are:
+
+Wednesday, October 26: 10:30 - 8:00 ET Thursday, October 27: 10:30 - 5:30 ET
+Friday, October 27: 10:30 - 4:00 ET
```
Comment (chalin) [inline content/en/blog/2022/kubecon-na.md:67]:
```suggestion
- Wednesday, October 26, 10:30 - 20:00 ET
- Thursday, October 27, 10:30 - 17:30 ET
- Friday, October 27, 10:30 - 16:00 ET
```

File: content/en/blog/2022/kubecon-na.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+We’re super excited to learn from all of you about your experience with
+OpenTelemetry, how you are using OpenTelemetry, features you may need and issues
+you may be running into when using OpenTelemetry.
+
+Come join us to listen, learn and get involved in OpenTelemetry.
+
+See you in Detroit in three weeks!
```
Comment (chalin) [inline content/en/blog/2022/kubecon-na.md:75]:
```suggestion
See you in Detroit!
```

File: content/en/blog/2022/kubecon-na.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+[eBPF Day](https://events.linuxfoundation.org/cloud-native-ebpf-day-north-america/)
+also has some OpenTelemetry talks focused on profiling and tracing using eBPF
+events. This event will be held on Tuesday, Oct 25 2022 and is co-located with
+Kubecon NA. Check out the event schedule
+[here](https://events.linuxfoundation.org/cloud-native-ebpf-day-north-america/program/schedule/).
+
+The **OpenTelemetry project meeting** is to meet and network with the
+OpenTelemetry maintainers and core contributors. This meeting will be held 3-5
+pm ET on Tuesday Oct 25, 2022 in Room 336 at the
```
Comment (chalin) [inline content/en/blog/2022/kubecon-na.md:55]:
```suggestion
OpenTelemetry maintainers and core contributors. This meeting will be held 3-5
PM ET on Tuesday Oct 25, 2022 in Room 336 at the
```

**Pull request merged**